dashboard/test_d_graphs.py

```diff
@@ -0,0 +1,188 @@
+# pylint skip-file
+"""Testing the d_graphs file"""
+import re
+import altair as alt
+import pytest
+from unittest.mock import patch
+import pandas as pd
+import datetime
```

Comment:
Small detail, but you could add a space between python libraries and functions you're importing from other scripts.